app/src/main/java/com/oracle/javafx/scenebuilder/app/DocumentWindowController.java

```diff
@@ -316,6 +316,13 @@ public enum ActionStatus {
             }
             event.consume();
         }
+        
+        // Keep preview function working even if preview accelerator is 
+        // already occupied in FXML
+        if ("p".equalsIgnoreCase(event.getText()) && modifierDown) {
```

Comment:
Can we replace this with `KeyCode.P.equals(event.getCode()) && modifierDown` ?

Reply:
Yes, I have replaced this expression with the proposal. Also tested it on macOS 12, Fedora 36 and Win10 and it works on all 3 platforms.